subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -333,6 +333,17 @@ Use the link:java_gradle_plugin.html[`java-gradle-plugin`]'s `validatePlugins` t
 
 Gradle changes the compression library used for creating archives from an Ant based one to https://commons.apache.org/proper/commons-compress/[Apache Commons Compress™]. As a consequence archives created from the same content, are unlikely to end up identical byte-by-byte to their older versions, created with the old library.
 
+==== Jar, War, and Ear destination directory defaults are set by JavaBasePlugin
+
+Previously, the `base` plugin would configure the
+link:{groovyDslPath}/org.gradle.api.tasks.bundling.AbstractArchiveTask.html#org.gradle.api.tasks.bundling.AbstractArchiveTask:destinationDirectory[destinationDirectory] of
+link:{groovyDslPath}/org.gradle.api.tasks.bundling.Jar.html[Jar], link:{groovyDslPath}/org.gradle.api.tasks.bundling.War.html[War], and
+link:{groovyDslPath}/org.gradle.plugins.ear.Ear.html[Ear] tasks to the directory specified by
+link:{groovyDslPath}/org.gradle.api.plugins.BasePluginExtension.html#org.gradle.api.plugins.BasePluginExtension:libsDirectory[BasePluginExtension#getLibsDirectory].
+In Gradle 8.0, this behavior now requires the `java-base` plugin to be applied. For projects which already apply the
+`java-base` plugin directly or indirectly through the `java`, `application`, `java-library`, or other JVM ecosystem plugins,
+this behavior should remain unchanged.
```

Comment:
```suggestion
==== Java Base Plugin now sets Jar, War, and Ear destination directory defaults 

Previously, the `base` plugin configured the
link:{groovyDslPath}/org.gradle.api.tasks.bundling.AbstractArchiveTask.html#org.gradle.api.tasks.bundling.AbstractArchiveTask:destinationDirectory[destinationDirectory] of
link:{groovyDslPath}/org.gradle.api.tasks.bundling.Jar.html[Jar], link:{groovyDslPath}/org.gradle.api.tasks.bundling.War.html[War], and
link:{groovyDslPath}/org.gradle.plugins.ear.Ear.html[Ear] tasks to the directory specified by
link:{groovyDslPath}/org.gradle.api.plugins.BasePluginExtension.html#org.gradle.api.plugins.BasePluginExtension:libsDirectory[BasePluginExtension.getLibsDirectory].
In Gradle 8.0, `java-base` handles this configuration.
No changes are required for projects that already apply the
`java-base` plugin directly or indirectly through the `java`, `application`, `java-library`, or other JVM ecosystem plugins.
```

Suggestions:

- make title more descriptive, readable
- replace octothorpe with period in link (typo, I *think*)
- improve readability of final sentence (qualifier list is quite long, better to lead with "no changes required" for scannability)

Consider adding another sentence at the end that describes what projects that *don't* already apply the `java-base` plugin ought to do. Should they apply the plugin directly? Is that situation even possible/probable? If not... maybe we don't even need this note?

Reply:
TIL the word "octothorpe".  `class#method` is familiar notation used in other places of the upgrade guide.

Reply:
I incorporated these changes.